### DISCUSSION TITLE
Feature Request: Select a column when adding an issue to a project

### DISCUSSION THREAD

**Discussion by joshdholtz** [community/community#2] [Mobile]
First of all... I ❤️ this app so much

## Feature Request
Even though I don't need to be able to access projects from the mobile app, I'll often assign an issue to a project. It would be great if I could also select which column I'd like to go to similar to this 👇 

<img width="447" alt="Screen Shot 2020-10-14 at 8 48 38 AM" src="https://user-images.githubusercontent.com/401294/95998034-1037a780-0dfa-11eb-8afe-e0c105e4b8fc.png">

**Comment by candyho** [comment]
Hey @joshdholtz , thanks for sharing your feedback.  Better in-app projects support is definitely something on our radar and we have been having conversations around the exact scenario you're describing! I will bring this request to the team again. Stay tuned and please keep the feedback coming. 💪

**Reply by joshdholtz** [reply]
>Stay tuned and please keep the feedback coming. 💪

😈😈😈😈😈😈😈😈😈

**Reply by heyaustinwood** [reply]
@candyho it would also be helpful if we could filter in /issues by project column 👍

**Reply by Igorko777** [reply]
https://github.com/community/community/discussions/2#discussioncomment-97982

**Reply by Igorko777** [reply]
https://github.com/community/community/discussions/2#discussioncomment-97982

**Comment by mbbischoff** [comment]
Better support for Projects is also my top request.

**Reply by ShadBalti** [reply]
Absolutely agree! Enhanced support for Projects would be a game-changer. 🚀

**Comment by 3kh0** [comment]
I still think it is crazy, we cant vote on #1 but we can vote on this one

**Comment by Mostafalarkilarki** [comment]
> First of all... I ❤️ this app so much
> 
> ## Feature Request
> Even though I don't need to be able to access projects from the mobile app, I'll often assign an issue to a project. It would be great if I could also select which column I'd like to go to similar to this 👇
> 
> <img alt="Screen Shot 2020-10-14 at 8 48 38 AM" width="447" src="https://user-images.githubusercontent.com/401294/95998034-1037a780-0dfa-11eb-8afe-e0c105e4b8fc.png">

**Comment by Igorko777** [comment]
https://github.com/orgs/community/discussions/2
